### PR TITLE
#657: New rewrite: expand import selectors

### PR DESCRIPTION
**Pull request by ysusuk**

### CODE REVIEW

File: core/src/test/resources/rewrite/ExpandImportSelectors.stat [hunk 1]
```diff
@@ -0,0 +1,34 @@
+rewrite.rules = [ExpandImportSelectors]
```
Comment (ysusuk) [inline core/src/test/resources/rewrite/ExpandImportSelectors.stat]: i most probably missed some corner casses! can you help me with them!

**Comment by olafurpg** [comment]
Wow. Thank you for this contribution!

Try renames.

Try running it on scala-repos (on my gh profile), start for example with
kafka.

**Comment by ysusuk** [comment]
@olafurpg tried on several repos. Looks good for me.

**Comment by olafurpg** [comment]
Can you share diffs from scala-repos?

**Comment by ysusuk** [comment]
[diffs.zip](https://github.com/olafurpg/scalafmt/files/707358/diffs.zip)

**Comment by olafurpg** [comment]
Awesome, great job! lgtm :+1:

**Comment by olafurpg** [comment]
will try to publish 5.2.0 soon, still struggling with hand pain

**Pull request merged**